src/features/ui/ToggleButton/ToggleButton.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC } from 'react';
+
+import styles from './ToggleButton.module.scss';
+
+type ToggleButtonProps = {
+	label?: string | React.ReactNode;
+	toggled: boolean;
+	hideOnOffLabels?: boolean;
+	onClick: () => void;
+};
+
+export const ToggleButton: FC<ToggleButtonProps> = ({
```

Comment:
We'll need to add a toggle button to zUI. For now I pulled this over from zns-dapp and made some minor changes.

Reply:
I've improved this component in Migrate Domain Settings task.